tools/cldr-code/src/main/java/org/unicode/cldr/test/SubmissionLocales.java

```diff
@@ -47,25 +77,37 @@ public final class SubmissionLocales {
         Set<String> temp = new HashSet<>(CLDR_OR_HIGH_LEVEL_LOCALES);
         temp.retainAll(GrammarInfo.getGrammarLocales());
         LOCALES_FOR_LIMITED = ImmutableSortedSet.copyOf(temp);
+
+        Set<String> temp2 = new HashSet<>(CLDR_LOCALES);
+        temp2.removeAll(SPECIAL_ORG_LOCALES);
+        TC_ORG_LOCALES = ImmutableSortedSet.copyOf(temp);
     }
 
     /**
      * New locales in this release, where we want to allow any paths even if others are restricted
      */
-    public static Set<String> ALLOW_ALL_PATHS = ImmutableSet.of(
-        "brx",
-        "ks",
-        "ks_Deva",
-        "rhg"   // Rohingya
+    public static Set<String> ALLOW_ALL_PATHS_BASIC = ImmutableSet.of(
+        // locales open for v43:
+        // "agp"
+        "lmo", // Lombardi
+        // "pap",
```

Comment:
I thought there were 4 Basic locales.

Reply:
Others not merged yet

Reply:
lmo and rif have been merged (and are in this list).
ajp and pap have not yet been merged.

@srl295 I think that "agp" in this list should be "ajp".

Reply:
will update to `ajp` once ajp is in

Reply:
updated.